crates/solidity/outputs/rust/tests/src/cst_output/generated/Expression.rs

```diff
@@ -0,0 +1,59 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use crate::cst_output::runner::run;
```

Comment:
I suggest adding the following tests, as they proved troublesome from debugging earlier:

- for `Expression` parser (treated as unary suffix?):
  - `foo ? true : false`
  - `(foo || bar) ? true : false`
- for `FunctionCallExpression` parser:
  - just `foo` (without parenthesis) should fail, instead of returning an `Identifier`

Reply:
The individual expression parsers need to be removed i.e. `FunctionCallExpression` because it only works by being inlined into the pratt mechanism.

Reply:
That's one of the polishing items I didn't get to.

Reply:
Aha, this is perhaps related to my `Fragment` suggestion we discussed f2f. We can put it aside for later.

Perhaps adding the top cases if you agree?

Reply:
Sure

Reply:
ternary expression is unary suffix, should be low precedence

Reply:
I'll add a lot more tests in a second PR to keep momentum before the retreat

Reply:
Thanks. Added the suggestions above in #299